src/components/views/elements/CopyableText.tsx

```diff
@@ -42,7 +44,12 @@ const CopyableText: React.FC<IProps> = ({ children, getTextToCopy }) => {
         }
     };
 
-    return <div className="mx_CopyableText">
+    let className;
+    if (border) {
+        className = "mx_CopyableText_border";
+    }
+
+    return <div className={classNames("mx_CopyableText", className)}>
```

Comment:
You should be able to do something like this:

```suggestion
    const className = classNames("mx_CopyableText", {
        mx_CopyableText_border: border
    });

    return <div className={className}>
```